src/es/components/molecules/tiles/Tiles.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+ * Tiles
+ * An example at: src/es/components/pages/Home.html
+ *
+ * @export
+ * @class Tiles
+ * @type {CustomElementConstructor}
+ */
+
+export default class Tiles extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderCSS () {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  }
+
+  renderCSS () {
+    this.css = /* css */`
+      :host {
+        padding-bottom: var(--component-spacing);
+      }
+      
+      :host .tiles__wrapper {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+      }
+
+      :host .tiles__tile {
+        width: 48%;
+        height: 350px;
+        margin-right: 1rem;
+        margin-bottom: 1rem;
```

Comment:
I would outsource the hard-coded units here into a separate variable. This will make it easier to adjust later. But since the component will certainly have to be tackled again, we can leave it as it is for the time being.